packages/lodestar-beacon-state-transition/src/fast/util/epochContext.ts

```diff
@@ -49,7 +49,8 @@ export class EpochContext {
   public currentShuffling!: IEpochShuffling;
   public nextShuffling!: IEpochShuffling;
   public config: IBeaconConfig;
-  public epochProcess?: IEpochProcess;
+  // validator balance at the time of epoch transition, not the one in state
+  public balances?: Gwei[];
```

Comment:
Are those justfied balances? Maybe rename it to justifiedBalances.
Comment above it confuses me even more haha

Reply:
it's not justified balance, it's balance when we process an epoch and prepare `epochProcess`. Maybe `checkpointBalance` or `epochBalance`?

once a checkpoint becomes justified, it becomes justified balance